### ISSUE TITLE
Very secretive Sybil group

### ISSUE THREAD

**Issue by tigerrrrrrrrrrrrrrrrr** [LayerZero-Labs/sybil-report#5606]
# Reported Addresses

0xad9b6b0a6cf5d8b273efd740fa24042bdcebef4f
0x0fce64deebe95c4a6a8f282a6882b895ba1cd843
0x02b040e1fbef33cdfce31d31dc62c2f5dfc60eaa
0x45678bf86abbf35d1c91d90028ccf4258d0d60cc
0x67a447c4567207991b3a7c58460ef19fa886c1a1
0xa6b4a3eb3ad8be4f52776788b01877713f39e88b
0xd00e2c85021d016e52849c167232e4de7e2d5dec
0x3eec65102ac4ec0f503d14a3c010b2466ab6658e
0x58a177769a7d53b73445798a65026adb38a17180
0x2640b7f63bdd8e9448a15032a8719c183541dfdd
0x26b988c8951a8021d97b33eb1b1d74d7becc96b2
0xfe7f05d509dcab3fb499a729b4573b6ee93d56ff
0x08b355510db1d45affe7f6fa342efcbff5053667
0x4e0b5a416c027e8efb0b70c7745ca6428bc710c8
0xfefc8e153faad97bb9946d8660b57e1e7792cddc
0xafaebf7b99207309ead303bcadb9674fb5925278
0x8a0f6d3635156de0ef4f7c711f48c9d8db2a3cf5
0x3030f96ed50219c2e7709eed2f71f2242d7b7194
0x762b4ab279e767826661bf01a7cda5b3e5e3533a
0x64d2899cfaf4cbfcf6f621fc8676bec00b07815d

# Description
I have repeatedly compared these addresses, combined with various on-chain data, from asset distribution, interaction time, interaction habits and other data comparisons, and determined that they come from a Sybil group

# Detailed Methodology & Walkthrough
method 1:
Enter all the addresses into the https://debank.com/ website and you can see that the creation times of these addresses are 2022-08-28 and 2021-08-20.
Then the asset distribution of all addresses is very similar, and the farming projects are almost the same, as you can see here, you can also open the above website and enter it yourself to view
![image](https://github.com/LayerZero-Labs/sybil-report/assets/170150396/a464be3d-7ba1-447a-a413-50313c5eda2a)
![image](https://github.com/LayerZero-Labs/sybil-report/assets/170150396/60ac017d-cca0-4a42-8ae6-1fc00e617d93)
![image](https://github.com/LayerZero-Labs/sybil-report/assets/170150396/b8cc5bc3-fd59-4fc3-8764-c8278b051c04)
![image](https://github.com/LayerZero-Labs/sybil-report/assets/170150396/b1ee554f-6a17-4931-8c79-555593cbd6db)

Method 2:
Now switch to the OP main network. They completed almost the same operation on 2023/03/20. All addresses generated 6 TXs on that day, of which some addresses had 5 TXs, and the times were very close.
All addresses completed interactions with the two addresses 0x93431a2e46023e0f37417304c357170b5615e138 and 0xc0507025958724ff877c9c76cc7b1e4c16cf1b67 on the two days of 2023/03/13 and 2023/03/12.
All addresses completed the interaction with the address 0x9aa90632bca10e2b24d69543d291f27848cf392d on 2023/03/09, and the time difference between the interactions was no more than 20 minutes. I compiled a few TXs at random. You can take a look.
https://optimistic.etherscan.io/tx/0x29faef019a64ca3255fdda19ecaf546e89458bd393c2e81c12c6ba9a0c3462e5
https://optimistic.etherscan.io/tx/0xf8dd0394ae515bde40d279aeb9791a9a7b179ad66b9814acfe06e60547980d15
https://optimistic.etherscan.io/tx/0xd077c93180156a662f68daf18d60603430c582819023910034c2dcc96c9067a6
You can also open https://debank.com/ and enter the address to view their data on the OP main network. I just listed some, and other behaviors are also highly similar.
https://debank.com/profile/0xad9b6b0a6cf5d8b273efd740fa24042bdcebef4f/history?chain=op
https://debank.com/profile/0x3030f96ed50219c2e7709eed2f71f2242d7b7194/history?chain=op
https://debank.com/profile/0x08b355510db1d45affe7f6fa342efcbff5053667/history?chain=op
https://debank.com/profile/0xfefc8e153faad97bb9946d8660b57e1e7792cddc/history?chain=op
I randomly picked out a few addresses above.
![image](https://github.com/LayerZero-Labs/sybil-report/assets/170150396/07f7d709-990c-4e8b-b0f7-556b981d0cb8)

Method 3:
Looking at the ETH main network, all addresses are between 2024/01/25 and 2024/01/28. They have completed the interaction with Kroma Bridge. The amounts are similar, all around 20U.
All addresses completed interaction with Manta Pacific Bridge on 2023/10/15, and the amount and time were similar, the amount was about 10U
All addresses completed interaction with Kroma Bridge on 2023/10/05, within one hour of each other.
All addresses completed the Starknet Bridge fund transfer between 2023/09/07 and 2023/09/11, and the transferred funds were about 1.5E
All addresses used Zora Bridge on 2023/08/30, and the times were very close, no more than an hour apart.
Most of the addresses used Zora Bridge on 2023/08/11, and the time was very close, with a difference of no more than one hour.
All addresses used Base Bridge on 2023/08/04, and the time is very close
All addresses used Zora Bridge on 2024/05/16
There are many similarities between these addresses on the ETH main network. In addition to the similarities I listed, there are many more. I randomly selected a few addresses for you to take a look at.
https://debank.com/profile/0xfefc8e153faad97bb9946d8660b57e1e7792cddc/history?chain=eth&token=
https://debank.com/profile/0xa6b4a3eb3ad8be4f52776788b01877713f39e88b/history?chain=eth&token=
https://debank.com/profile/0x0fce64deebe95c4a6a8f282a6882b895ba1cd843/history?chain=eth&token=
https://debank.com/profile/0xad9b6b0a6cf5d8b273efd740fa24042bdcebef4f/history?chain=eth&token=

![image](https://github.com/LayerZero-Labs/sybil-report/assets/170150396/e2a6f729-0e18-494d-9d7a-b9b87cbc549f)
![image](https://github.com/LayerZero-Labs/sybil-report/assets/170150396/be1d234f-84d7-4495-aa42-68018ca9b016)

Method 4:
View layerzeroscan.com data directly
The total number of TXs for all addresses is very similar, all around 70 TXs.
All addresses completed interaction with Star Protocol on 2023-06-07 and 2023-06-08. You can directly enter the address at layerzeroscan.com to view
All addresses completed interaction with Abracadabra Money on June 17, 2023, and the time is very close. You can directly enter the address at layerzeroscan.com to view
All addresses completed interaction with Holograph on July 13, 2023, and the time is very close. You can directly enter the address at layerzeroscan.com to view
These addresses have many similarities in the TX that the L0 interacts with. In addition to the ones I mentioned, there are many more. You can check it out. I randomly selected a few addresses.
https://layerzeroscan.com/address/0xad9b6b0a6cf5d8b273efd740fa24042bdcebef4f
https://layerzeroscan.com/address/0x45678bf86abbf35d1c91d90028ccf4258d0d60cc
https://layerzeroscan.com/address/0xfe7f05d509dcab3fb499a729b4573b6ee93d56ff
https://layerzeroscan.com/address/0x8a0f6d3635156de0ef4f7c711f48c9d8db2a3cf5
https://layerzeroscan.com/address/0x64d2899cfaf4cbfcf6f621fc8676bec00b07815d

![image](https://github.com/LayerZero-Labs/sybil-report/assets/170150396/4cc88d75-1c2d-4605-9ff9-b1d0c976a17c)

To sum up, there are more than 10 similarities that I listed, and there are many more similarities. By comparison, it can be determined that they are from the same Sybil group, and they are masters of disguise.

# Reward Address (If Eligible)
0x9351301ea0c16b39d59d38f97f98f9a66c62c66c
